### ISSUE TITLE
White space below cell in /edit page

### ISSUE THREAD

**Issue by rsmith31415** [ipython/ipython#8040]
I was a bit surprised by the presentation of the `/edit` page. Is it supposed to look like this:

![screenshot from 2015-03-12 20 37 51](https://cloud.githubusercontent.com/assets/599274/6632203/f2f26234-c8f7-11e4-9ced-e234d9f6b08d.png)

As you see, there is a huge white space below the cell. I'm not sure if that is intentional.

**Comment by Carreau** [comment]
Wait, is the white part of your comment, or screenshot ? Where does the browser stops ?

![capture d ecran 2015-03-12 a 21 14 00](https://cloud.githubusercontent.com/assets/335567/6632960/c261c510-c8fc-11e4-92a6-c025dc9f5090.png)

**Comment by rsmith31415** [comment]
No, the image I posted is the full screenshot. Your screenshot shows how it should look. 

I installed a clean environment to verify that I was not mistaken. Let me try again. Maybe I had caching issues.

**Comment by rsmith31415** [comment]
Well, that was interesting. I had to restart the computer and create a new anaconda environment to get the correct behavior. This happened across anaconda environments. Now everything looks okay.
